src/Faker/Provider/pt_BR/PhoneNumber.php

```diff
@@ -14,12 +14,21 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
     protected static $cellphoneFormats = array('9####-####');
 
     /**
-     * Generates a 2-digit area code not composed by zeroes.
+     * Pick one random entry out of an array of area codes.
```

Comment:
Not fan of the new description. It's a still generator function, so it should be something like

> Generates a valid 2-digit area code